README.md

```diff
@@ -35,14 +35,37 @@ Changes For Ras-OAuth2-Server
 This documents changes implemented for the Ras-OAuth2-Server that allow this solution to be used with the microservice
 framework at ONS.
 
-Admin User
+Admin API
 ----------
 
-The admin user has been added to the SQLite DB, which is part of the repo.
+The OAuth2 server now has an admin interface. This has it's own REST endpoint to allow a client with the correct
+client_id and client_secret to interact with the Interface.
 
-Username: admin
-Email: admin@email.com
-Password: admin2017
+The endpoint is at /api/account/create and accepts [POST], but will accept a [GET] and query parameters.
+
+* Mandatory Parameters
+
+HTTP Basic Authentication  username: password
+username:       This should be the username to populate the users of the OAuth2 server. It should be in an email format and will be checked to be unique
+password:       This is a password for that user.
+client_id:      This is the client_id making the request to use the admin interface API. This should be the same as the user in the authentication header
+client_secret:  This is the password of the client_id.
+
+```
+
+curl -X POST http://localhost:8040/api/account/create/ -u ons@ons.gov:password -d 'username=testuser4@email.com&password=password&client_id=ons@ons.gov@client_secret=password'
```

Comment:
Client ID needs to be ons@ons.gov.uk. However, we need to automate the insert of 'Authentication and Authorisation Users' and 'Client Identifiers for Oauth2 Server' so that u:ons@ons.gov.uk p:password is environment specific and aligns for the client MS and the Oauth2 server (e.g. ras_frontstage and ras_party have their own client Id and password rather than both using ons@ons.gov.uk | password).